src/resolve/resolved-etc-hosts.c

```diff
@@ -164,7 +165,7 @@ int etc_hosts_parse(EtcHosts *hosts, FILE *f) {
         unsigned nr = 0;
         int r;
 
-        for (;;) {
+        for (nr = 0; ; nr++) {
```

Comment:
hmm, this means line numbering now starts at zero, right? this is probably not what most people expect. Also nr is assigned zero a few lines up already, one should probably go